release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -3595,4 +3595,27 @@ $ oc adm release info 4.8.49 --pullspecs
 [id="ocp-4-8-49-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-8-50"]
+=== RHSA-2022:6511 - {product-title} 4.8.50 bug fix update
+
+Issued: 2022-09-21
+
+{product-title} release 4.8.50 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:6511[RHSA-2022:6511] advisory. There are no RPM packages with this update.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.8.50 --pullspecs
+----
+
+[id="ocp-4-8-50-bug-fixes"]
+==== Bug fixes
+* Previously, operators had to delete and recreate the IngressController to enable proxy protocol for clusters updated to 4.8. With this update, setting the IngressController API status fields `spec.endpointPublishingStrategy.hostNetwork.protocol` or `spec.endpointPublishingStrategy.nodePort.protocol` to `PROXY` updates the fields on upgraded clusters. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2084337#[*BZ#2084337*])
```

Comment:
```suggestion
* Previously, operators had to delete and recreate the IngressController to enable the PROXY protocol for clusters updated to 4.8. With this update, setting the IngressController API status fields `spec.endpointPublishingStrategy.hostNetwork.protocol` or `spec.endpointPublishingStrategy.nodePort.protocol` to `PROXY` updates the fields on upgraded clusters. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2084337#[*BZ#2084337*])
```

Reply:
It seems to be PROXY: https://docs.openshift.com/container-platform/4.8/networking/ingress-operator.html#nw-ingress-controller-configuration-proxy-protocol_configuring-ingress